src/tess_sip/tess_sip.py

```diff
@@ -92,65 +99,101 @@ def SIP(
             period_at_max_power: the best fit period of the sinusoid.
             power_bkg: the power at each period for the pixels -outside- the aperture
             raw_lc_bkg: the background light curve (pixels outside aperture)
+            model: the systematics model used to correct the light curve
     """
 
-    # Get the un-background subtracted data
-    if hasattr(tpfs[0], "flux_bkg"):
-        tpfs_uncorr = [
-            (tpf + np.nan_to_num(tpf.flux_bkg))[
-                np.isfinite(np.nansum(tpf.flux_bkg, axis=(1, 2)))
+    if ((type(tpfs) is lk.collections.TargetPixelFileCollection) or 
+            (type(tpfs) is lk.targetpixelfile.TessTargetPixelFile)):
+        # Get the un-background subtracted data
+        print("Target Pixel File Input")
```

Comment:
We don't want print statements in the code, because they're hard to quiet for a user. If you want a log message, we can using the `logging` package. In this case, I don't think it's useful

Reply:
I forgot to delete these lines when I was testing the code. :)